THIRD-PARTY-LICENSES.txt

```diff
@@ -0,0 +1,386 @@
+     (Apache License, Version 2.0) Apache Commons BeanUtils (commons-beanutils:commons-beanutils:1.9.4 - https://commons.apache.org/proper/commons-beanutils/)
+     (Apache License, Version 2.0) Apache Commons Codec (commons-codec:commons-codec:1.13 - https://commons.apache.org/proper/commons-codec/)
+     (Apache License, Version 2.0) Apache Commons Collections (commons-collections:commons-collections:3.2.2 - http://commons.apache.org/collections/)
+     (The Apache Software License, Version 2.0) Apache Commons Configuration (commons-configuration:commons-configuration:1.10 - http://commons.apache.org/configuration/)
+     (The Apache Software License, Version 2.0) Commons Digester (commons-digester:commons-digester:1.8.1 - http://commons.apache.org/digester/)
+     (Apache License, Version 2.0) Apache Commons IO (commons-io:commons-io:2.6 - http://commons.apache.org/proper/commons-io/)
+     (The Apache Software License, Version 2.0) Commons Lang (commons-lang:commons-lang:2.6 - http://commons.apache.org/lang/)
+     (The Apache Software License, Version 2.0) Apache Commons Logging (commons-logging:commons-logging:1.2 - http://commons.apache.org/proper/commons-logging/)
+     (The Apache Software License, Version 2.0) Commons Net (commons-net:commons-net:3.3 - http://commons.apache.org/proper/commons-net/)
+     (Apache License, Version 2.0) Apache Commons Validator (commons-validator:commons-validator:1.6 - http://commons.apache.org/proper/commons-validator/)
+     (Unknown license) dom4j (dom4j:dom4j:1.6.1 - http://dom4j.org)
+     (MIT) refined (eu.timepit:refined_2.12:0.9.4 - https://github.com/fthomas/refined)
+     (Apache 2.0) Circe generic extras (io.circe:circe-generic-extras_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) Circe generic (io.circe:circe-generic_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) Circe java8 (io.circe:circe-java8_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) Circe jawn (io.circe:circe-jawn_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) Circe literal (io.circe:circe-literal_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) Circe parser (io.circe:circe-parser_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) Circe refined (io.circe:circe-refined_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) Circe shapes (io.circe:circe-shapes_2.12:0.11.1 - https://github.com/circe/circe)
+     (Apache 2.0) circe-yaml (io.circe:circe-yaml_2.12:0.9.0 - https://github.com/circe/circe-yaml)
+     (Unknown license) cwlavro-generated (io.cwl:cwlavro-generated:2.0.4.3 - no url defined)
+     (Unknown license) cwlavro-tools (io.cwl:cwlavro-tools:2.0.4.3 - no url defined)
+     (GNU General Public License (GPLv3)) dockstore-client (io.dockstore:dockstore-client:1.8.0-beta.1 - https://github.com/dockstore/dockstore-cli/dockstore-client)
```

Comment:
Doesn't seem right, either this is being detected wrong or we should change these to Apache 2

Reply:
Fixed this so Apache 2.0 is the license; another pull request at dockstore-cli will fix the license information for dockstore-client, dockstore-file-plugin-parent, and openapi-java-wes-client

Reply:
Still looking into how to specify license information for '-generated' artifacts

Reply:
Oh, we actually have access to cwlavro-generated, it is https://github.com/common-workflow-language/cwlavro

Reply:
I can now add license information for artifacts previously labeled 'Unknown license' but somehow the original labels 'Unknown license' and 'no url defined' don't go away. E.g:

(The Apache Software License, Version 2.0) (Unknown license) cwlavro-generated (io.cwl:cwlavro-generated:2.0.4.3 - no url defined)

Reply:
According to documentation https://www.mojohaus.org/license-maven-plugin/examples/example-thirdparty.html that is expected result